site/content/in-dev/unreleased/realm.md

```diff
@@ -0,0 +1,57 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+Title: Entities
```

Comment:
This might be a miss on my end... Originally, I did not put `Realm` in this set of docs because technically, there is no Realm _entity_ in Polaris. However, it's still a conceptual entity, so I think having it here makes sense.

Reply:
This PR created a new page for realm only. Two options:
1. Move the realm related doc to the entity page here, https://polaris.apache.org/in-dev/unreleased/entities/
2. Rename the title to `Realm` instead of `Entities`

I'm OK with either one, but prefer option 1 to make the doc compacted.

Reply:
Oh, yeah this looks wrong. @fabio-rizzo-01 can you fix the title or move the content?